pkgs/development/python-modules/paintcompiler/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  fonttools,
+  black,
+}:
+
+buildPythonPackage rec {
+  pname = "paintcompiler";
+  version = "0.3.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "simoncozens";
+    repo = "paintcompiler";
+    rev = "v${version}";
+    hash = "sha256-dmVBQUUyFc71zq8fXBQ4ii/llrGdtUCOponCzSeut6g=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    fonttools
+    black
```

Comment:
Why do we need a formatter?

Reply:
Because it can decompile COLR tables to python files and format these with black.